app/lib/json.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * Safely parse a JSON string.
+ * @returns The parsed JSON object or null if parsing fails
+ */
+export const safeJsonParse = (jsonString: string): unknown | null => {
+  try {
+    return JSON.parse(jsonString);
+  } catch {
+    return null;
```

Comment:
I am not sure if we can return null because null can also be a valid result if you pass string 'null'. should we maybe return `{ success: true, value: T } | { success: false }` instead?

Reply:
done, and changed validation in the guest-account-storage to use zod